README.md

```diff
@@ -7,12 +7,12 @@
 
 ## Screenshot
 
-![screenshot](path)
+![screenshot](assets/images/web-pic.png)
 
 
 ## Built with
 - Major languages: HTML, CSS, JavaScript
-- Technologies used: Github, Linters
+- Technologies used: SASS, Github, Linters
 
 ## Getting Started
 Clone the repository and setup the linters
```

Comment:
- [ ] OPTIONAL: Please do not forget to paste actual link to the project 😉 :
![image](https://user-images.githubusercontent.com/77038610/152218451-61c6aefb-d8ce-4c8c-a077-f9e0e3bce64e.png)